private/dependency_tree_parser.bzl

```diff
@@ -91,6 +91,13 @@ def _generate_imports(repository_ctx, dep_tree, explicit_artifacts, neverlink_ar
         elif repository_ctx.attr.fetch_sources and ":sources:" in artifact["coord"]:
             # We already processed the sources above, so skip them here.
             pass
+        elif ":json:" in artifact["coord"]:
+            seen_imports[target_label] = True
+            versioned_target_alias_label = "%s_extension" % escape(artifact["coord"])
+            all_imports.append(
```

Comment:
hmm, seems weird to generate a target for a non-jar/aar/jvm package dependency. I can see this being confusing. Let's just skip generating targets for coords containing the `:json:` packaging?

I believe all you want to do is to fetch jar artifacts without the resolution breaking on artifacts with json dependencies, right?

Reply:
Generic maven server supports arbitrary package types, I'd inflate other types as regular file targets
You never know what people download from maven haha

(ie OkBuck downloads Buck binary as `pex` package from maven)

Reply:
Right, sounds good, since the json artifact is marked as the primary artifact in `artifact["file"]`.